ams_app/templates/ams_app/dashboard.html

```diff
@@ -0,0 +1,149 @@
+      <div class="pagination-div">
+        <ul class="pagination">
+          {% if assignments.has_previous %}
+            <li class="waves-effect"><a href="?page={{ assignments.previous_page_number }}"><i class="material-icons">chevron_left</i></a></li>
+          {% else %}
+            <li class="disabled"><span><i class="material-icons">chevron_left</i></span></li>
+          {% endif %}
+          {% for i in assignments.paginator.page_range %}
+            {% if assignments.number == i %}
+              <li class=" waves-effect active"><span>{{ i }} <span class="sr-only">(current)</span></span></li>
+            {% else %}
+              <li class="waves-effect"><a href="?page={{ i }}">{{ i }}</a></li>
+            {% endif %}
+          {% endfor %}
+          {% if assignments.has_next %}
+            <li class="waves-effect"><a href="?page={{ assignments.next_page_number }}"><i class="material-icons">chevron_right</i></a></li>
+          {% else %}
+            <li class="disabled"><span><i class="material-icons">chevron_right</i></span></li>
+          {% endif %}
+        </ul>
+      </div>
+      <div class="no-assignment">
+          <table class="striped responsive-table table">
+          <thead>
+            <tr>
+                <th>#</th>
+                <th>Title</th>
+                <th>Upload</th>
+                <th>Date Created</th>
+                <th>Due date</th>
+                <th>Course code</th>
+                <th>Course title</th>
+                <th>Submissions</th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {% for assignment in assignments %}
+            <tr>
+              <td>{{ forloop.counter }}</td>
+              <td><a href="/assignments/{{ assignment.id }}">{{ assignment.title }}</a></td>
+              <td>{{ assignment.upload }}</td>
+              <td>{{ assignment.created_at }}</td>
+              <td>{{ assignment.due_date }}</td>
+              <td>{{ assignment.course_code }}</td>
+              <td>{{ assignment.course_title }}</td>
+              <td><a href="/assignments/{{ assignment.id }}/submissions">{{ assignment.submission_set.count }}</a></td>
+            </tr>
+            {% endfor %}
+          </tbody>
+        </table>
+      </div>
+    </div>
+  </div>
+  <div>
+
+{% else %}
+  <div class="body">
+    <div class="sub-body">
+      <div class="row">
+        <div class="col s2">
+          <a class="btn-floating btn-large waves-effect waves-light red add-assignment modal-trigger" href="#modal1"><i class="material-icons">add</i></a>
+        </div>
+      <div class="no-assignment">
+        <p> No assignment created yet </p>
+      </div>
+    </div>
+  </div>
+{% endif %}
+
+<!-- Modal Structure -->
+  <div id="modal1" class="modal">
+    <div class="modal-content">
+      <h4>Create Assignment</h4>
+      <div>
+        <div class="modal-container">
+          <form method="POST" action="/assignments/add" enctype="multipart/form-data">
```

Comment:
Let's adopt using static urls rather than hardcoding the url
You used it here "ams_app/templates/ams_app/pass.html"